pkg/sdk/deps_builder.go

```diff
@@ -31,12 +32,17 @@ func (sdk *poktrollSDK) buildDeps(
 	// Supply the logger
 	deps = depinject.Configs(deps, depinject.Supply(polylog.Ctx(ctx)))
 
+	cosmosClient, err := cosmosclient.NewClientFromNode(config.QueryNodeUrl.Host)
```

Comment:
```suggestion
	cosmosClient, err := cosmosclient.NewClientFromNode(config.QueryNodeUrl.String())
```

If only .Host is passed the cosmosclient will default the scheme to http